OracleGoldenGate/21c/README.md

```diff
@@ -34,12 +35,18 @@ For more information about Oracle GoldenGate please see the [Oracle GoldenGate 2
 Once you have downloaded the Oracle GoldenGate software, a container image can be created using the Docker command-line interface.
 A single `--build-arg` is needed to indicate the GoldenGate installer which was downloaded.
 
-```sh
-$ docker build --tag=oracle/goldengate:21.3.0.0.0 \
-                --build-arg INSTALLER=213000_fbo_ggs_Linux_x64_Oracle_services_shiphome.zip .
-Sending build context to Docker daemon
-...
-Successfully tagged oracle/goldengate:21.3.0.0.0
+To create a container image for GoldenGate for Oracle, use the following script:
```

Comment:
Revert this change and use the example for Oracle as a template for the BigData example.

Changes to markdown should pass 'markdown-lint'

Reply:
Done.
In addition to this change, there were other markdown-lint corrections in this file suggested by this lint:  https://github.com/markdownlint/markdownlint